src/io/ioexternalbus.cpp

```diff
@@ -4,13 +4,30 @@
 #include <QPainter>
 #include <QPen>
 
+#include <QtWidgets>
```

Comment:
I dont think this is the right include; this include pulls in _all_ Qt widget headers, as far as i remember

Reply:
Yes, you are right. Replaced.